### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by mattburgess**

### CODE REVIEW

File: tasks/auth.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@
     owner: root
     mode: 0640
   with_items: "{{ concourse_local_users }}"
+  loop_control:
+    label: "{{ item.user }}"
```
Comment (ahelal) [inline tasks/auth.yml]: awesome that is great ?

File: tasks/common_nix.yml [hunk 2]
```diff
@@ -1,10 +1,15 @@
 ---
 
 
-- name: common | Combine dictionary options
+- name: common | Combine dictionary options (web)
```
Comment (ahelal) [inline tasks/common_nix.yml]: Any reason why you split them ?

Comment (mattburgess) [reply tasks/common_nix.yml]: From the 2nd commit in the series:

concourse_facts_local_users is set in tasks/auth.yml which is only
included on web nodes; therefore the combining step now fails on
worker-only nodes as that var is empty.

**Comment by ahelal** [comment]
sorry for the delay was a little busy.

**Pull request merged**